src/assets/text/authors.js

```diff
@@ -0,0 +1,82 @@
+export default {
+    // do not delete section. delete individuals as needed. modify content as needed
+    // currently only 'fullName', 'firstName', 'initials', 'profile_link', and 'contribution' are used
+    primaryAuthors: [
```

Comment:
SC: When I made `primaryAuthors: []` blank, the page still ran. I wonder if an error or alert would be better than it still running, since it's supposed to always be primary author 1+? Otherwise, it just does this: 

<img width="734" alt="image" src="https://user-images.githubusercontent.com/19958841/201709875-2bed758d-f924-44a8-8fe8-6ae2255ba6dd.png">

Reply:
ooh, good catch

Reply:
K just added a catch. Now _no_ authors are listed if no `primaryAuthors` are specified.